output-html.inc

```diff
@@ -205,6 +205,9 @@ function output_html($search_result, $is_admin=FALSE) {
         }
         $entry["workdaylocation"] = '<ul><li>' . htmlspecialchars($workday_location) . '</li></ul>';
     }
+    if(isset($entry["workdaytimezone"])){
+        $entry["workdaylocation"] .= '<ul class="clock"><li>' . htmlspecialchars($entry["workdaytimezone"]) . '</li></ul>';
```

Comment:
If you don't mind the small amount of extra work renaming it here and in the CSS, I'd like to call this "timezone" rather than "clock" to match the field being displayed.